docs/source/reference/config.rst

```diff
@@ -1111,6 +1114,40 @@ Kueue configuration (optional).
 
 Name of the `local queue <https://kueue.sigs.k8s.io/docs/concepts/local_queue/>`_ to use for SkyPilot jobs.
 
+.. _config-yaml-kubernetes-dws:
+
+``kubernetes.dws``
```

Comment:
This option should only be allowed when the `autoscaler` is set to `gke`? We should have the check?

Reply:
Added check.